src/Scrutor/IImplementationTypeFilter.cs

```diff
@@ -135,5 +135,12 @@ public interface IImplementationTypeFilter : IFluentInterface
         /// <param name="predicate">The predicate to match types.</param>
         /// <exception cref="ArgumentNullException">If the <paramref name="predicate" /> argument is <c>null</c>.</exception>
         IImplementationTypeFilter Where(Func<Type, bool> predicate);
+
+        /// <summary>
+        /// Filter out types where the namespace is not or a child of the provided one.

```

Comment:
`where the namespace is not or a child of the provided one`? What's the purpose of this method compared to `InNamespaces`?